components/core/MinioHandler.py

```diff
@@ -0,0 +1,24 @@
+from DownloadDaemon import *
+from minio import Minio
+from minio.error import ResponseError
+
+client = Minio('localhost:9000', secure=False,
+               access_key='bassa',
+                  secret_key='bassa123')
+
+def upload_to_minio(file_name, file_path):
+	try:
+		bucket_status = client.bucket_exists('bassa')
+		#print(bucket_status)
+		if(bucket_status == False):
```

Comment:
@demon-36 shall we use `not` operator here, which seems to be synonymous with previous code style.
```
If not bucket_status:
```